azerothcore/azerothcore-wotlk
# Fix(Core/Quest): For The Horde!

## Changes Proposed:
-  Added RP and buff.
-  Implemented two methods to cast a certain spell to all players in a map, zones or areas.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/1690

## SOURCE:

Positions from VMangos.

## Tests Performed:

- Tested in-game

## How to Test the Changes:


1. .q a 4974
2. .q c 4974
3. Turn in the quest.
4. Place other chars on Razor Hill, Crossroads and Camp Taurajo.
5. You should see the yell and get the buff.

## Known Issues and TODO List:


- [ ] There is a missing Herald of Thrall for Durotar zone (so you can see the yell), but I couldn't find it.
